mavenExample/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>ua.hillel.javaqaauto</groupId>
+    <artifactId>mavenExample</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>18</maven.compiler.source>
+        <maven.compiler.target>18</maven.compiler.target>
+        <testng.version>7.8.0</testng.version>
+        <maven.surefire.version>3.1.2</maven.surefire.version>
+    </properties>
+
+
+    <profiles>
+      <profile>
+          <id>dev</id>
+          <dependencies>
+              <dependency>
+                  <groupId>org.testng</groupId>
+                  <artifactId>testng</artifactId>
+                  <version>${testng.version}</version>
+                  <scope>test</scope>
+              </dependency>
+
+          </dependencies>
+
+          <build>
+              <plugins>
+                  <plugin>
+                      <groupId>org.apache.maven.plugins</groupId>
+                      <artifactId>maven-surefire-plugin</artifactId>
+                      <version>${maven.surefire.version}</version>
+                      <configuration>
+                          <systemPropertyVariables>
+                              <url>http://dev.google.com</url>
+                          </systemPropertyVariables>
+                          <!--                    <suiteXmlFiles>-->
+                          <!--                        <suiteXmlFile>testng.xml</suiteXmlFile>-->
+                          <!--                    </suiteXmlFiles>-->
```

Comment:
Please, do not leave commented parts of code. It makes no sense and it's a bad practice.
```suggestion                        
```